modules/47_regipol/regiCarbonPrice/bounds.gms

```diff
@@ -79,9 +79,10 @@ loop(regi$(sameAs(regi,"DEU")),
 
 *** only small amount of co2 injection ccs until 2030 in Germany
 vm_co2CCS.up(t,regi,"cco2","ico2",te,rlf)$((t.val le 2030) AND (sameas(regi,"DEU"))) = 1e-3;
-*** no Pe2Se fossil CCS in Germany, if cm_noPeFosCCDeu = 1 chosen 
-vm_emiTeDetail.up(t,regi,peFos,enty,te,"cco2")$((sameas(regi,"DEU")) AND (cm_noPeFosCCDeu = 1)) = 1e-4;
-
+*** no Pe2Se fossil CCS in Germany, if c_noPeFosCCDeu = 1 chosen 
```

Comment:
Cool, so you put all your hard-coded Germany references in regipol? That will please Lavinia! Do you only need these bounds in case you have regional emissions target and regipol = ``regiCarbonPrice``? If you also need it in BAU runs you would also need these bounds in the ``none`` realization of regipol.